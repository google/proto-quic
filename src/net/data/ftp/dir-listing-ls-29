.:
total 1204984
drwxrwxrwx   2 sunle    sunle       2048 Apr  6 00:36 .
drwxrwxrwx 9406 sunle    sunle     245760 Apr 10 18:00 ..
-rw-rw----   1 ftp      sunle    110583757 Apr  5 13:43 GRBRATLIN64_4_77_ia64_4.77_S0@874222@03717.rpm.Z
-rw-rw----   1 ftp      sunle    121591811 Apr  5 13:43 GRBRATLIN_4_77_ia32_4.77_S0@874222@20701.rpm.Z
-rw-rw----   1 ftp      sunle    110965625 Apr  5 13:43 XENRATLIN64_4_77_ia64_4.77_S0@874222@17785.rpm.Z
-rw-rw----   1 ftp      sunle    121986153 Apr  5 13:44 XENRATLIN_4_77_ia32_4.77_S0@874222@09635.rpm.Z
-rw-rw----   1 ftp      sunle    151525255 Apr  5 13:44 XENRATS10_4_77_sun4_10_4.77_S0@874222@21358.dstream.Z
