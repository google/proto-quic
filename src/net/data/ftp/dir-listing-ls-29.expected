d
.
-1
1994
4
6
0
36

d
..
-1
1994
4
10
18
0

-
GRBRATLIN64_4_77_ia64_4.77_S0@874222@03717.rpm.Z
110583757 
1994
4
5
13
43

-
GRBRATLIN_4_77_ia32_4.77_S0@874222@20701.rpm.Z
121591811 
1994
4
5
13
43

-
XENRATLIN64_4_77_ia64_4.77_S0@874222@17785.rpm.Z
110965625 
1994
4
5
13
43

-
XENRATLIN_4_77_ia32_4.77_S0@874222@09635.rpm.Z
121986153 
1994
4
5
13
44

-
XENRATS10_4_77_sun4_10_4.77_S0@874222@21358.dstream.Z
151525255 
1994
4
5
13
44
